### PR TITLE
Bump monitoring setup manifests

### PR DESCRIPTION
**Pull request by brancz**
Super-seeds #294 

With these changes everything should work except for the number of Pods graph on the nodes pages, as this is a change that has to be done in console/bridge.

@s-urbaniak @alexsomesan @fabxc

### CODE REVIEW

**Comment by s-urbaniak** [comment]
No objections from my side if we manage to bump the console until the release ;)

**Comment by sym3tri** [comment]
@brancz what needs to be changed in Console?

**Comment by s-urbaniak** [comment]
@sym3tri a hint was given in https://github.com/coreos-inc/tectonic-issues/issues/232#issuecomment-297080919

File: modules/tectonic/resources/manifests/monitoring/prometheus-k8s.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  namespace: tectonic-system
+  labels:
+    prometheus: k8s
+spec:
+  replicas: 1
+  version: ${prometheus_version}
+  serviceAccountName: prometheus-k8s
+  serviceMonitorSelector:
+    matchExpression:
+    - {key: k8s-apps, operator: Exists}
```
Comment (chancez) [inline modules/tectonic/resources/manifests/monitoring/prometheus-k8s.yaml]: So this means that if someone else wanted promtheus to monitor their app, they would need to have a label `k8s-apps`, even if its not a k8s app?

**Comment by brancz** [comment]
Thanks for the quick and smooth collaboration everyone! :tada:

**Pull request merged**